bolt-sidecar/src/crypto/bls.rs

```diff
@@ -124,4 +125,94 @@ mod tests {
         let sig = blst::min_pk::Signature::from_bytes(signature.as_ref()).unwrap();
         assert!(signer.verify(&msg, &sig, &pubkey));
     }
+
+    fn random_constraints(count: usize) -> Vec<FullTransaction> {
+        // Random inclusion request
+        let json_req = r#"{
+            "slot": 10,
+            "txs": ["0x02f86c870c72dd9d5e883e4d0183408f2382520894d2e2adf7177b7a8afddbc12d1634cf23ea1a71020180c001a08556dcfea479b34675db3fe08e29486fe719c2b22f6b0c1741ecbbdce4575cc6a01cd48009ccafd6b9f1290bbe2ceea268f94101d1d322c787018423ebcbc87ab4"]
+        }"#;
+
+        let req: InclusionRequest = serde_json::from_str(json_req).unwrap();
+
+        (0..count).map(|_| req.txs.first().unwrap().clone()).collect()
+    }
+
+    #[tokio::test]
+    async fn generate_test_data() {
```

Comment:
I understand why you created this test and it looks good, however I would suggest moving it out of the `bls` module. Not fully sure where yet, but here it's a bit confusing IMO

Reply:
yeah I agree. I think `src/test_util.rs` might be a good place? I just thought of keeping this because will be helpful in future

Reply:
Sure, perfect

Reply:
done